src/Microsoft.OData.Client/DataServiceClientFormat.cs

```diff
@@ -267,7 +267,7 @@ internal IEdmModel LoadServiceModelFromNetwork()
                     context.UsePostTunneling,
                     requestEventArgs.Headers);
 
-                httpRequest = new HttpWebRequestMessage(args);
+                httpRequest = new HttpClientRequestMessage(args);

```

Comment:
How come we are instatiating a `HttpClientRequestMessage` here? I assumed the `TransportMode` property would be used to determine whether to create a `HttpClienRequestMessage` or `HttpWebRequestMessage`. Or better yet, some factory function that returns an `IODataRequestMessage`

Reply:
resolved verbally.